addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ExternalRedirectScanRule.java

```diff
@@ -81,17 +83,17 @@ public class ExternalRedirectScanRule extends AbstractAppParamPlugin {
     /** The various (prioritized) payload to try */
     private static final String[] REDIRECT_TARGETS = {
         REDIRECT_SITE,
-        HttpHeader.HTTP + "://" + REDIRECT_SITE,
-        HttpHeader.HTTPS + "://" + REDIRECT_SITE,
-        HttpHeader.HTTPS + "://" + REDIRECT_SITE.replace(".", "%2e"), // Double encode the dots
+        HttpHeader.SCHEME_HTTP  + REDIRECT_SITE,
+        HttpHeader.SCHEME_HTTPS + REDIRECT_SITE,
+        HttpHeader.SCHEME_HTTPS + REDIRECT_SITE.replace(".", "%2e"), // Double encode the dots
         "5;URL='https://" + REDIRECT_SITE + "'",
-        HttpHeader.HTTP + ":\\\\" + REDIRECT_SITE,
-        HttpHeader.HTTPS + ":\\\\" + REDIRECT_SITE,
+        "URL='http://" + REDIRECT_SITE + "'",
+        HttpHeader.SCHEME_HTTP + "\\" + REDIRECT_SITE,
+        HttpHeader.SCHEME_HTTPS + "\\" + REDIRECT_SITE,
         "//" + REDIRECT_SITE,
         "\\\\" + REDIRECT_SITE,
         "HtTp://" + REDIRECT_SITE,
         "HtTpS://" + REDIRECT_SITE,
-        "URL='http://" + REDIRECT_SITE + "'",
```

Comment:
Re-ordered this so that a meta location payload is actually tried at MEDIUM strength.